src/main/java/by/exadel/internship/service/impl/FileServiceImpl.java

```diff
@@ -37,6 +35,7 @@ public String upload(byte[] fileContent, String originalFileName) {
         MDCLog.putClassNameInMDC(SIMPLE_CLASS_NAME);
         String fileName = originalFileName;
         fileName = UUID.randomUUID().toString()
+                .concat(".")
```

Comment:
You already have constant for this (even in this class)

Reply:
Fixed